build_aarch64_wheel.py

```diff
@@ -18,11 +18,10 @@
 
 # AMI images for us-east-1, change the following based on your ~/.aws/config
 os_amis = {
-    'ubuntu18_04': "ami-0f2b111fdc1647918",  # login_name: ubuntu
     'ubuntu20_04': "ami-0ea142bd244023692",  # login_name: ubuntu
```

Comment:
Not sure why are you removing ubutnu18-04
```suggestion
    'ubuntu18_04': "ami-0f2b111fdc1647918",  # login_name: ubuntu
    'ubuntu20_04': "ami-0ea142bd244023692",  # login_name: ubuntu
```

Reply:
Hi @malfet, even for the manylinux2014 docker based builds, we are still dependent on the host OS for gfortran workaround in the script at:
https://github.com/pytorch/builder/blob/main/build_aarch64_wheel.py#L444

and to support ubuntu 18.04 host build (one of the supported options in this script), it needs lot of tool updates which probably not worth going forward. 

> pytorch library builds require python >=3.7 and cmake >=3.13. The default
> versions of python and cmake on ubuntu 18.04 are much older. Given ubuntu 18.04
> EOL is April 2023, the effort to update the script for ubutnu 18.04 changes
> is not worth, hence deprecating ubuntu 18.04 host os support.